packages/frontend/app/components/locale-chooser.js

```diff
@@ -67,8 +66,8 @@ export default class LocaleChooserComponent extends Component {
     }
   }
   @action
-  clearFocus() {
-    const buttons = this.menuElement.querySelectorAll('button');
+  clearFocus(event) {
+    const buttons = event.target.parentElement.children;
```

Comment:
This still gives me an error:
![Screenshot 2024-07-10 at 2 36 17 PM](https://github.com/ilios/frontend/assets/2374391/4aa6f188-cdbc-470c-b7b1-f355aca963d7)